### ISSUE TITLE
Please add this to nuget

### ISSUE THREAD

**Issue by Shadowblitz16** [JulNadeauCA/libagar#44]
Please add this to nuget.
Its very hard to follow your visual studio installation instructions as they are meant for really old versions of visual studio.

if anything there needs to be a set of instructions for newer versions like visual studio 2017 and 2019

**Comment by JulNadeauCA** [comment]
Thanks for the suggestion, I'll look into nuget as soon as I get a chance.

The installation instructions have been tested with both VS 2015 and 2017. Are there issues with VS 2017 that I'm not aware of, or that the guide covered incorrectly? I don't have a copy of VS 2019 so I'm unable to test that myself right now.

**Comment by Shadowblitz16** [comment]
I am using visual studio 2017, however the is no Tools / Options / Projects and Solutions / VC++ Directories / Show directories for option.
![image](https://user-images.githubusercontent.com/2453707/68891832-83123d00-06d6-11ea-95a9-a4c84f787380.png)

from what I have learned is you have to go under project settings to do this now.
